docs/client/data.js

```diff
@@ -5716,7 +5716,17 @@ DocsData = {
     "memberof": "Mongo.Cursor",
     "name": "count",
     "options": [],
-    "params": [],
+    "params": [
+      {
+        "description": "<p>Optional. Specifies whether to consider the effects of the <code>skip</code> and <code>limit</code> options. By default, the <code>count()</code> method ignores the effects of the <code>skip</code> and <code>limit</code> options. Set <code>applySkipLimit</code> to true to consider the effect of these options.</p>",
```

Comment:
It's documented that `count()` works completely differently on the server and on the client? On the client, it acts as if `applySkipLimit` was true, on the server as if false.

This is what this PR attempts to resolve. Wasn't sure about the docs, guess I shouldn't have touched them in the first place.